src/client/src/app/store/reducers/user/user.reducer.ts

```diff
@@ -24,16 +24,15 @@ export const initialState: UserState = {
   pRightName: '',
   pLeftSocketId: '',
   pRightSocketId: '',
-}; 
+};
 
 
 export const reducer = createReducer(
   initialState,
 
   on(signInSuccess, (state, action) => {
+    localStorage.setItem('token', JSON.stringify(action.data))
```

Comment:
you should add a better name for the item you are setting .. token is misleading ... maybe userToken? or even just user ... you call it token then im pretty sure you are setting it to an object that is a partial user

Reply:
done!